### PR TITLE
Uts for orgs

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

File: src/storage/mongodb/SiteAreaStorage.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ export default class SiteAreaStorage {
           {
             $set: {
               siteAreaID: DatabaseUtils.convertToObjectID(siteArea.id),
-              siteID: DatabaseUtils.convertToObjectID(siteArea.siteID)
+              siteID: DatabaseUtils.convertToObjectID(siteArea.siteID),
+              companyID: DatabaseUtils.convertToObjectID(siteArea.site?.companyID)
```
Comment (ClaudeROSSI) [inline src/storage/mongodb/SiteAreaStorage.ts:49]: I do not understand why we need to add companyID here.

The query is done using { withSite: true } so the information is already available, right?
>> siteArea.site?.companyID

File: src/server/rest/v1/service/AuthorizationService.ts [hunk 2]
```diff
@@ -554,7 +554,7 @@ export default class AuthorizationService {
         'site.id', 'site.public', 'site.name', 'site.address', 'siteID', 'voltage', 'coordinates', 'forceInactive', 'manualConfiguration', 'firmwareUpdateStatus', 'tariffID',
         'capabilities', 'endpoint', 'chargePointVendor', 'chargePointModel', 'ocppVersion', 'ocppProtocol', 'lastSeen',
         'firmwareVersion', 'currentIPAddress', 'ocppStandardParameters', 'ocppVendorParameters', 'connectors', 'chargePoints',
-        'createdOn', 'chargeBoxSerialNumber', 'chargePointSerialNumber', 'powerLimitUnit'
+        'createdOn', 'chargeBoxSerialNumber', 'chargePointSerialNumber', 'powerLimitUnit', 'companyID'
```
Comment (ClaudeROSSI) [inline src/server/rest/v1/service/AuthorizationService.ts:557]: why don't we expose 'site.companyID' instead?

File: src/storage/mongodb/SiteAreaStorage.ts [hunk 3]
```diff
@@ -66,7 +67,8 @@ export default class SiteAreaStorage {
           {
             $set: {
               siteAreaID: null,
-              siteID: null
+              siteID: null,
+              companyID: null
```
Comment (ClaudeROSSI) [inline src/storage/mongodb/SiteAreaStorage.ts:71]: idem ... this should not be necessary

**Review by ClaudeROSSI — CHANGES_REQUESTED** [review]
please have a look to my comments.

BTW: automated tests are failing. Can you check why?

**Comment by LucasBrazi06** [comment]
@ClaudeROSSI Could you check the latest Mahdi's changes?

**Review by ClaudeROSSI — APPROVED** [review]
@mahdibenromdhane: your code was correct.

My latest comments can be ignored. After verifying again with Serge we could confirm that the siteArea does not include the companyID and this is normal.

**Pull request merged**